content/references/rippled-api/peer-crawler.md

```diff
@@ -0,0 +1,86 @@
+# Peer Crawler
+
+The Peer Crawler is a special API endpoint for reporting on the health and topology of the peer-to-peer network. This API method is available by default on a non-privileged basis through the [Peer Protocol](peer-protocol.html) port, which is also used for `rippled` servers' peer-to-peer communications about consensus, ledger history, and other necessary information.
+
+The information reported by the peer crawler is effectively public, and can be used to report on the overall XRP Ledger network, its health, and topology.
+
+## Request Format
+
+To request the Peer Crawler information, make the following HTTP request:
+
+- **Protocol:** https
+- **HTTP Method:** GET
+- **Host:** (any `rippled` server, by hostname or IP address)
+- **Port:** (the port number where the `rippled` server uses the Peer Protocol, typically 51235)
+- **Path:** `/crawl`
+- **Security:** Most `rippled` servers use a self-signed certificate to respond to the request. By default, most tools (including web browsers) flag or block such responses for being untrusted. You must ignore the certificate checking (for example, if using cURL, add the `--insecure` flag) to display a response from those servers.
+
+**Tip:** Since this request uses the GET method, you can test this request using just the URL bar of your web browser. For example, <https://s1.ripple.com:51235/crawl> requests peer crawler information from one of Ripple's public servers.
+
+## Response Format
+
+The response has the status code **200 OK** and a JSON object in the message body.
+
+The JSON object has the following fields:
+
+| `Field`          | Value  | Description                                      |
+|:-----------------|:-------|:-------------------------------------------------|
+| `counts`         | Object | _(May be omitted)_ Stats about this server's health, similar to the response from the [get_counts method][]. The default configuration does not report this field. Information reported includes: how large the ledger and transaction databases are, the cache hit rate for the in-application caches, and how many objects of various types are cached in memory. Types of objects that may be stored in memory include ledgers (`Ledger`), transactions (`STTx`), validation messages (`STValidation`), and more. |
+| `overlay` | Object  | Information about the peer servers currently connected to this one, similar to the response from the [peers method][]. Contains one field, `active`, which is an array of objects (see below). |
```

Comment:
All of these fields are actually configurable iirc. You could choose to hide `overlay` but still provide `unl` So the "May be omitted" applies to all the fields (except `version`, which should always be there).